sdk/testing/src/main/java/io/opentelemetry/sdk/testing/assertj/HistogramPointAssert.java

```diff
@@ -74,6 +75,20 @@ public HistogramPointAssert hasBucketBoundaries(double... boundaries) {
     return this;
   }
 
+  /**
+   * Asserts the {@code boundaries} field matches the expected value. The boundaries are may vary
+   * with a specified precision.
+   *
+   * @param boundaries The set of bucket boundaries in the same order as the expected collection.
+   * @param precision The precision under which boundaries may vary.
+   */
+  public HistogramPointAssert hasBucketBoundaries(double[] boundaries, Offset<Double> precision) {
+    isNotNull();
+    double[] actualBoundaries = actual.getBoundaries().stream().mapToDouble(d -> d).toArray();
+    Assertions.assertThat(actualBoundaries).as("boundaries").containsExactly(boundaries, precision);
```

Comment:
Maybe we change the behavior of `hasBucketBoundaries(double... boundaries)` to include some default reasonable precision.

Reply:
👍 
How about 0.000001?